extensions/wikia/AdEngine/js/provider/directGpt.js

```diff
@@ -34,6 +34,7 @@ define('ext.wikia.adEngine.provider.directGpt', [
 			},
 			INCONTENT_BOXAD_1:          {size: '300x250,160x600,300x600', loc: 'middle'},
 			INCONTENT_PLAYER:           {size: '1x1', 'loc': 'middle', 'pos': 'incontent_player'},
+			INCONTENT_LEADERBOARD:      {size: '728x90,300x250', 'loc': 'hivi', 'pos': 'incontent_leaderboard'},
```

Comment:
By default `pos` key gets value based on slot name ([source](https://github.com/Wikia/app/blob/95a4f0b11882e84a580e7742f2fc78c55bcce069/extensions/wikia/AdEngine/js/provider/factoryWikiaGpt.js#L75)) - do we need here to lowercase this value? I think we can't. It won't work with Rubicon Fastlane. RPFL creative requires the same pos value as defined in fastlane module ([here](https://github.com/Wikia/app/pull/9777/files#diff-83eb1fa9c5afed5b4b7d3f13ec99171aR140)) so there is a difference (lowercase vs uppercase)